dev-packages/node-integration-tests/suites/esm/warn-esm/test.ts

```diff
@@ -5,7 +5,7 @@ afterAll(() => {
 });
 
 const esmWarning =
-  '[Sentry] You are using Node.js in ESM mode ("import syntax"). The Sentry Node.js SDK is not compatible with ESM in Node.js versions before 18.19.0 or before 20.6.0. Please either build your application with CommonJS ("require() syntax"), or use version 7.x of the Sentry Node.js SDK.';
```

Comment:
Snuck this in. It's probably not in our interest to point people towards v7 any longer.